### PR TITLE
Use new lists: $file_extensions_common_archives and $file_extensions_macros

### PR DESCRIPTION
**Pull request by jkamdjou**
Also in this PR:

- Updated some legacy Unsolicited snippets
- Updated the RDP file rule severity to medium (h/t @ajpc500)
- Renamed 3 files for consistency (below for reference): @rw-access should we add the feed ID so they don't show up as deleted/new rules when viewing the Feed? That would also keep historically flagged data on these rules in tact. If yes, I could use a reminder on how to do that
- Small FP update

```
 rename detection-rules/{attachment_any_html_suspicious_javascript_identifiers.yml => attachment_html_smuggling_rot13.yml}
 rename detection-rules/{attachment_html_smuggling.yml => attachment_html_smuggling_unescape.yml}
 rename detection-rules/{office_remote_html_template.yml => attachment_office_remote_html_template.yml}
```

### CODE REVIEW

**Review by ianthiel — APPROVED** [review]
beautiful 👍

**Comment by rw-access** [comment]
to preserve the rule ID, you need to manually specify the ID in the YAML so that it matches whatever it was before. it's generated deterministically from the hash of the path.

also keep in mind that lists only are synced when bora boots and also nightly. since we don't have many guarantees about bora booting I would wait until tomorrow to merge

**Comment by jkamdjou** [comment]
@rw-access adding rule IDs worked perfectly, confirmed the 3 rules show up as `Updated` instead of `Deleted`+`New` in a test branch

File: detection-rules/impersonation_wells_fargo.yml
Comment (rw-access) [inline detection-rules/impersonation_wells_fargo.yml:12]: no substrings anymore? sounds fine to me, otherwise you may want `strings.icontains`

Comment (jkamdjou) [reply detection-rules/impersonation_wells_fargo.yml:12]: yeah no substring anymore, too many FPs

**Review by rw-access — APPROVED** [review]

**Comment by aidenmitchell** [comment]
I believe this query will also need to be updated with the $file_extensions_macros list.
https://github.com/sublime-security/sublime-rules/blob/ed14fdf1208707fa1ecd47bce9a1eec6f0e5843a/queries/attachments/vba_macros.yml#L4

**Comment by jkamdjou** [comment]
> I believe this query will also need to be updated with the $file_extensions_macros list.
> 
> https://github.com/sublime-security/sublime-rules/blob/ed14fdf1208707fa1ecd47bce9a1eec6f0e5843a/queries/attachments/vba_macros.yml#L4

NICE @aidenmitchell thank you

**Pull request merged**
